live_fixtures.gemspec

```diff
@@ -27,4 +27,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "temping", "~> 3.0"
   spec.add_development_dependency "byebug"
   spec.add_development_dependency "sqlite3"
+  spec.add_development_dependency "yard"
```

Comment:
I'm not 100% sure, but here's what I think:

YARD is a gem that analyzes this gem's codebase and automatically creates documentation based on the code and its comments. [[more info]](http://www.rubydoc.info/gems/yard/file/README.md)

By adding it as a development dependency, it ensures that YARD will be available to future developers of the gem when they want to run the YARD server.

Your question reminds me that the docs should probably have people run YARD using `bundle exec`, so I'm glad you asked!

Maybe you or other people have more information about what this line does to share here.